source/agora/consensus/Ledger.d

```diff
@@ -1005,6 +994,16 @@ public class Ledger
 /// The Ledger class held by a node
 public class NodeLedger : Ledger
```

Comment:
It would be better to document the roles of the `NodeLedger` in more detail and so is for `Ledger` because we need to know the difference of their roles explicitly, which might be applied in later PRs, I guess.

Reply:
Good point, the documentation is severely lacking. I will try to improve it after the split.